app/build.gradle

```diff
@@ -136,10 +136,11 @@ dependencies {
     compile "com.android.support.test.espresso:espresso-idling-resource:$rootProject.espressoVersion"
 
     //Play Services
+    compile "com.google.android.gms:play-services-auth:$rootProject.playServicesVersion"
```

Comment:
Why do you need auth? It is not mentioned in the firebase integration documentation?